changelog/1769.feature.rst

```diff
@@ -0,0 +1,2 @@
+Refactor of `~plasmapy.formulary.collisions` to split the module
+into sub packages and have the ability to implement new functionality
```

Comment:
```suggestion
Refactored `~plasmapy.formulary.collisions`.  The file
:file:`collisions.py` was converted into a subpackage (directory) and
it's contents was split into appropriately categorized and named
sub-modules (files).
```

A little rewording to be consistent with our changelog guidelines and to be a little more explicit.